src/notes/FluxNotesEditor.jsx

```diff
@@ -225,7 +225,6 @@ class FluxNotesEditor extends React.Component {
         }
         let shortcut = this.props.newCurrentShortcut(shortcutC, shortcutTrigger, updatePatient);
         if (!Lang.isNull(shortcut) && shortcut.needToSelectValueFromMultipleOptions()) {
-//            console.log(text);

```

Comment:
Can we remove this console.log properly @nicoleng12

Reply:
What do you mean by properly?

Reply:
It was functionally removed by adding it and commenting it out. Can we remove it entirely if it's not necessary. Sorry if that was unclear

Reply:
I don't think the commented console.log is in the actual file.  There used to be a commented console.log that is removed in this PR.

Reply:
I"m an idiot. You're 100% right. Never let me comment on anything ever again. -_-

Reply:
No problem lol, I can see how you can think it was just commented out.  I thought there was a different way of removed console.logs instead of just deleting the line :D